app/src/androidTest/java/net/skyscanner/backpack/BpkSnapshotTest.kt

```diff
@@ -133,6 +136,36 @@ open class BpkSnapshotTest {
     scenario.close()
   }
 
+  protected fun ComposeTestRule.setScreenshotContent(
+    size: IntSize = IntSize(width, height),
+    background: Color = Color.Unspecified,
+    vararg providers: ProvidedValue<*>,
+    content: @Composable () -> Unit,
+  ) {
+    Assume.assumeFalse(BpkTestVariant.current == BpkTestVariant.Themed)
+
+    val asyncScreenshot = prepareForAsyncTest()
+
+    val scenario = launchActivity<AppCompatActivity>()
```

Comment:
the compose test rule already has launched an activity - does this need to be launched again?